packages/dd-trace/src/config.js

```diff
@@ -83,7 +88,6 @@ class Config {
     this.reportHostname = String(reportHostname) === 'true'
     this.scope = platform.env('DD_CONTEXT_PROPAGATION') === 'false' ? scopes.NOOP : scope
     this.clientToken = clientToken
-    this.componentName = componentName
```

Comment:
what's the reason of removing this as this is needed later to create a reporter tag `lightstep.component_name`

Reply:
component_name is equivalent to the datadog service configuration option. Python is using it, so we should do the same for consistency (or convince them to change).